src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -450,16 +450,24 @@ public OpenIdConnectHandler(HttpClient backchannel, HtmlEncoder htmlEncoder)
                 // Authorization Code or Hybrid flow
                 if (!string.IsNullOrEmpty(authorizationResponse.Code))
                 {
-                    // TODO: Does this event provide any value over AuthorizationResponseReceived or AuthorizationResponseValidated?
+                    // TODO: RedeemingAuthorizationCode?
```

Comment:
Yep. I'm pretty sure I've mentioned it in a previous PR, but `AuthorizationCodeReceived` makes no sense as `AuthorizationResponseReceived` already covers the main use cases.

`RedeemingAuthorizationCode` is much more appropriate name.